src/components/controls/Checkbox.tsx

```diff
@@ -4,7 +4,7 @@ import * as s from './checkbox.scss';
 
 interface ICheckboxProps {
     checked: boolean;
-    text: string;
+    text: React.ReactNode;
```

Comment:
Sure it can.
React.ReactNode is translated to: ReactElement | string | number | fragment | boolean | null | undefined.
So, it should be able to render all the things we need it for. I usually set all content props as React.ReactNode, since it can render both components and text.

Reply:
Okay, misunderstood your message. Will change the name to content